tex/rules.tex

```diff
@@ -543,6 +560,15 @@ \subsection{Agility \label{ref-agility}}
     \label{fig:robot_in_goal}
 \end{figure}
 
+\subsection{ Avoidance \label{ref-avoidance}}
+
+In order to be able to detect each other reliably, all robots must have 
+an avoidance ring. The avoidance ring is a 2.54 cm high hollow cylinder surrounding
+the robot. When viewed from top, this cylinder forms a convex hull of the robot shape.
+It is installed at the height of 12 cm  - 14.54 cm from the field floor.
+Its color must be the same as the color of the ball.
```

Comment:
Is there any specific reason for this?

Furthermore, what sort of a color should robots that play with the IR ball use?